actions/websocket_actions.jsx

```diff
@@ -1223,6 +1228,8 @@ function handleChannelViewedEvent(msg) {
 
 export function handlePluginEnabled(msg) {
     const manifest = msg.data.manifest;
+    dispatch({type: ActionTypes.RECEIVED_WEBAPP_PLUGIN, data: manifest});
```

Comment:
Is this change still needed?

Reply:
@larkox I kept this here since it's a bug fix